recipes/libffi/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "3.2.1" %}
+
+package:
+  name: libffi
+  version: {{ version }}
+
+source:
+  fn: libffi-3.2.1.tar.gz
+  url: ftp://sourceware.org/pub/libffi/libffi-3.2.1.tar.gz
+  sha1: 280c265b789e041c02e5c97815793dfc283fb1e6
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+
+test:
+  commands:
+    - true
```

Comment:
Some comment I made in the other recipes. (We have three easy merges waiting for a simple test :wink:)